toolkits/landcover/test/int/composites.int.test.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var lct = require('../../api.js');
+var TestImage = require('../helpers/test-image.js');
+
+withEarthEngine('Composites', function() {
+  it('createTemporalComposites()', function(done) {
+    var testCollection = ee.ImageCollection([
+      TestImage.create({value: 100}, '2015-12-01'),
+      TestImage.create({value: 150}, '2015-12-15'),
+      TestImage.create({value: 200}, '2016-01-01'),
+      TestImage.create({value: 250}, '2016-01-15'),
+      TestImage.create({value: 300}, '2016-02-01'),
+      TestImage.create({value: 350}, '2016-02-15'),
+      TestImage.create({value: 400}, '2016-03-01'),
+      TestImage.create({value: 450}, '2016-03-15'),
+      TestImage.create({value: 500}, '2016-04-01'),
+      TestImage.create({value: 550}, '2016-04-15')
+    ]);
+
+    var result = lct.Composites.createTemporalComposites(
+        testCollection,
+        /* startDate= */ '2016-01-01',
```

Comment:
Ugh.  This (commenting arguments to id them) is why we use pass by dictionary throughout the client library.
We need to get that working in here; I hate this idiom.

Reply:
Agreed. Filed #38 to track.